packages/cli-repl/src/format-output.ts

```diff
@@ -11,6 +12,57 @@ type EvaluationResult = {
   type?: string;
 };
 
+const i = Symbol.for('nodejs.util.inspect.custom');
```

Comment:
The idea behind this is so us to setup a custom inspect function specifically only for mongodb types. I chose to do this here, rather than at a higher level (internal state, service provider etc), because I wanted other implementors to receive data as is, and choose what to do with it in their own way. This is perhaps flawed, as it might be better for implementors to just receive raw bson instead.